src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+final class ReleaseCommand extends AbstractCommand
+{
+    private static $labels = array(
+        'patch' => 'blue',
+        'bug' => 'red',
+        'docs' => 'yellow',
+        'minor' => 'green',
+        'pedantic' => 'cyan',
+    );
+
+    private static $stabilities = array(
+        'patch' => 'blue',
+        'minor' => 'green',
+        'irrelevant' => 'yellow',
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $help = <<<'EOT'
+The <info>release</info> command analyzes pull request of a given project to determine
+the changelog and the next version to release.
+
+Usage:
+
+<info>php dev-kit release doctrine-orm-admin-bundle</info>
+
+The command will show what is the status of the project, then a list of pull requests
+made against selected branch (default: <info>3.x</info>) with the following information: stability, name,
+labels, changelog, url.
+
+After that, it will show what is the next version to release and the changelog for that release.
+EOT;
+
+        $this
+            ->setName('release')
+            ->setDescription('Helps with a project release.')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project to release.')
```

Comment:
what about making this also interactive and having autocomplete byes on the `projects.yml` ❓